Build fixes. gcc9, no submodules

proller opened the pull request:
Insignificant

proller committed:
Fix building without submodules

proller committed:
Fix more gcc9 warnings

alexey-milovidov commented on libs/libcommon/cmake/find_jemalloc.cmake:
```diff
@@ -30,7 +30,7 @@ if (ENABLE_JEMALLOC)
     if (JEMALLOC_LIBRARIES)
         set (USE_JEMALLOC 1)
-    else ()
+    elseif (NOT MISSING_INTERNAL_JEMALLOC_LIBRARY)
```
Excessive modification.

proller committed:
was wrong!
../dbms/src/IO/WriteBufferAIO.cpp:277:54: error: result of comparison 'ssize_t' (aka 'long') > 9223372036854775807 is always false [-Werror,-Wtautological-type-limit-compare]
    if ((static_cast<ssize_t>(flush_buffer.offset()) > std::numeric_limits<off_t>::max()) ||
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

proller committed:
Merge remote-tracking branch 'upstream/master' into fix25

proller commented:
now warning in clang:
../dbms/src/IO/WriteBufferAIO.cpp:277:54: error: result of comparison 'ssize_t' (aka 'long') > 9223372036854775807 is always false [-Werror,-Wtautological-type-limit-compare]
    if ((static_cast<ssize_t>(flush_buffer.offset()) > std::numeric_limits<off_t>::max()) ||
         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
